packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -321,6 +321,10 @@ function useMediaUploadQueue() {
 
           const trackTiming = getTimeTracker('load_upload_media');
 
+          if (resource.type !== 'image' && resource.isMuted !== null) {
```

Comment:
Why not `resource.type === 'video'`?

We have other types like `gif` too (those are always muted).

Reply:
Yeah and we are setting isMuted on gifs. That is what we want no?

Reply:
GIFs are always considered to be muted. We don't need to set `isMuted` on GIFs.